### PR TITLE
Support logical Types in Avro Export

### PR DESCRIPTION
**Pull request by jpraetorius**
- Map Datacontracts date-type fields to avro logical types
- date: `int/date`
- timestamp, timestamp_tz: `long/timestamp-millis`
- timestamp_ntz: `long/local-timestamp-millis`

Captures the state of discussion for #141

### CODE REVIEW

**Comment by simonharrer** [comment]
LGTM

**Comment by simonharrer** [comment]
can you add a changelog entry as well?

**Comment by jpraetorius** [comment]
✅ Done

File: tests/fixtures/avro/export/orders_with_datefields.avsc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "namespace": "com.example.checkout",
+  "fields": [
+    {
+      "name": "orderdate",
+      "doc": "My Field",
+      "type": "int",
+      "logicalType": "date"
+    },
+    {
+      "name": "order_timestamp",
```
[the remark this answers is not in the document]
Comment (jochenchrist) [reply tests/fixtures/avro/export/orders_with_datefields.avsc:14]: You are correct, this is a bug.

[the remark this answers is not in the document]
Comment (jochenchrist) [reply tests/fixtures/avro/export/orders_with_datefields.avsc:14]: https://github.com/datacontract/datacontract-cli/issues/233

**Pull request merged**